.gitlint.yaml

```diff
@@ -89,7 +89,7 @@ pylint:
     - .py
   command: pylint
   arguments:
-    - --rcfile={DEFAULT_CONFIGS}/pylintrc
+    - --rcfile=.configs/pylintrc
```

Comment:
You stole this :)

Reply:
Well it's the fix you did in the Subprocess PR :)